```
/nix/store/x5s5b448m5s9i1hi8n4r24c2f0b0310y-python3.8-xdis-5.0.11/lib/python3.8/site-packages/xdis/op_imports.py:173: in get_opcode_module
    return op_imports[canonic_python_version[vers_str]]
E   KeyError: '3.8.12'
```

Wow, that is something...

--- a/pkgs/development/python-modules/uncompyle6/default.nix
+++ b/pkgs/development/python-modules/uncompyle6/default.nix
@@ -12,12 +12,12 @@
 
 buildPythonPackage rec {
   pname = "uncompyle6";
-  version = "3.7.4";
+  version = "3.8.0";
   disabled = pythonAtLeast "3.9"; # See: https://github.com/rocky/python-uncompyle6/issues/331
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "af8330861bf940e7a3ae0f06d129b8e645191a36bf73ca15ff51997a837d41f8";
+    sha256 = "sha256-YgYzYY9t/B8+eBh+Igk014/8Y5wOOdrsofxTWquBcBQ=";
   };
 
   checkInputs = [ nose pytest hypothesis six ];

--- a/pkgs/development/python-modules/xdis/default.nix
+++ b/pkgs/development/python-modules/xdis/default.nix
@@ -6,14 +6,14 @@
 
 buildPythonPackage rec {
   pname = "xdis";
-  version = "5.0.11";
+  version = "6.0.2";
   disabled = isPy27;
 
   src = fetchFromGitHub {
     owner = "rocky";
     repo = "python-xdis";
     rev = version;
-    sha256 = "sha256-KTPu0+bERLRCVESqJgBPtcftlniWl2+C9GDcf84ssiA=";
+    sha256 = "sha256-P8mUkAO3usFCE+E9cna2x1iA2uyHVPX9FHDpX+kTFWQ=";
   };
 
   checkInputs = [ pytest ];

